internal/processor/processor.go

```diff
@@ -98,6 +106,52 @@ func (p *Processor) ProcessMetric(envelope *loggregator_v2.Envelope) {
 	}
 }
 
+// ProcessLog takes an envelope, parses it and sends the processed logs to the nozzle
+func (p *Processor) ProcessLog(envelope *loggregator_v2.Envelope) {
+	var err error
+	var logsMessage logs.LogMessage
+
+	var appTags []string
+	var serviceName string = envelope.SourceId
+	var source string
+
+	cfapp := p.appCache.Get(envelope.SourceId)
+	if cfapp != nil {
+		appTags = cfapp.Tags
+		appTags = append(appTags, fmt.Sprintf("application_id:%s", envelope.GetTags()["app_id"]))
+		appTags = append(appTags, fmt.Sprintf("application_name:%s", envelope.GetTags()["app_name"]))
+		appTags = append(appTags, fmt.Sprintf("instance_index:%s", envelope.GetTags()["instance_id"]))
+		for _, tag := range appTags {
+			if strings.HasPrefix(tag, "source") {
+				source = strings.TrimPrefix(tag, "source:")
+				break
+			}
```

Comment:
could we allow applications to specify that they want to send their logs to Datadog by including a special tag like `enable_dd_logs:true`? Reason being is that it is common for companies to charge back usage to teams and not all teams want to send logs or they may only want to send logs for some environments. Giving control via a tag or app env var would allow decision to send logs at the app level.

Reply:
Done! For now, it's enabled by default, unless explicitly disabled by `dd_logs_enabled: false`.